cranelift/filetests/filetests/isa/x64/run-const.clif

```diff
@@ -0,0 +1,13 @@
+test run
+target x86_64 use_new_backend
+
+function %test_compare_i32() -> b1 {
+block0:
+    v0 = iconst.i32 42
+    v1 = iconst.i32 42
+    v2 = icmp eq v0, v1
+    return v2
+}
+
+; TODO fixme!
+; r u n
```

Comment:
The reason for which it was "commented out" here is that this test doesn't pass yet with the new x64 backend: we need enough IR lowering for creating the trampolines (and iconst/icmp).